modules/templates/helper.go

```diff
@@ -371,7 +371,7 @@ func NewFuncMap() []template.FuncMap {
 		"RenderLabels": func(labels []*models.Label) template.HTML {
 			html := `<span class="labels-list">`
 			for _, label := range labels {
-				html += fmt.Sprintf("<div class='ui label' style='color: %s; background-color: %s'>%s</div>",
+				html += fmt.Sprintf("<div class='ui label' style='color: %s; background-color: %s'>%s</div> ",
```

Comment:
Is this space necessary?

Reply:
Yes, otherwise the labels would have no margin in between. Previously this was done using flexbox and margin but after this change with `display: inline`, whitespace between tags becomes significant again.

Having that whitespace also brings the benefit when selecting text for copy, it will select `label1 label2` instead of `label1label2`.